nixos/modules/services/networking/dhcpcd.nix

```diff
@@ -210,7 +210,8 @@ in
       { description = "DHCP Client";
 
         wantedBy = [ "multi-user.target" ] ++ lib.optional (!hasDefaultGatewaySet) "network-online.target";
-        wants = [ "network.target" ];
+        wants = [ "network.target" ] ++ lib.optional useResolvConf "resolvconf.service";
+        after = lib.optional useResolvConf "resolvconf.service";
```

Comment:
These shouldn't need to be optional since `resolvconf.service` just won't exist if disabled.

Reply:
Seems ok, done.